### PR TITLE
Add project infrastructure overview

### LINKED ISSUE #4: Document "infrastructure"
List of "infrastructure" we use and short description of it and how it "administer".

- GitHub
- GitHub actions
- CirrusCI
- Zapier
- Zulip
- DockerHub
- Netlify
- 1Password
- Zoom
- Cloudsmith
- forward.io
- Linode

and whatever else that should be on the list

### PR DESCRIPTION
**Pull request by SeanTAllen**
Closes #4

### CODE REVIEW

File: infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+Access to GitHub is divided into three different teams.
+
+* Contributor
+
+Members have [triage level](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) access to all projects.
+
+* Committer
+
+Member have [maintainer level](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) access to all projects.
```
Comment (EpicEric) [inline infrastructure.md]:
```suggestion
Members have [maintainer level](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) access to all projects.
```

**Review by EpicEric — CHANGES_REQUESTED** [review]

**Comment by EpicEric** [comment]
We also have a [PyPI](https://pypi.org) account, currently used to upload new versions of the [mkdocs theme](https://pypi.org/project/mkdocs-ponylang/). Currently me and @mfelsche have the credentials (that I know of); these should probably also be in the 1Password account.

**Comment by SeanTAllen** [comment]
@EpicEric suggestion added. Do you want to add the necessary pypi section to this PR?

File: infrastructure.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+## Netlify
+
+Netlify hosts a number of our websites including [the pony website](https://ponylang.io), [the pony tutorial](https://tutorial.ponylang.io), and [the pony patterns book](https://patterns.ponylang.org).
+
+Access is via individual accounts which can be granted various levels of access to different projects based on need.
+
+## Open Collective
+
+We have an account with [Open Collective](https://opencollective.com/ponyc) by which we can accept donations to support Pony development.
+
```
Comment (EpicEric) [inline infrastructure.md:102]:
```suggestion

## PyPI

We have a [PyPI](https://pypi.org) account, used to upload packaged releases of our [Mkdocs theme](https://github.com/ponylang/mkdocs-theme).

Access information is stored in the 1Password account.

```

**Comment by SeanTAllen** [comment]
@EpicEric pypi suggestion added. needs a final approval from you.

**Review by EpicEric — APPROVED** [review]

**Pull request merged**